test_start.py

```diff
@@ -1,25 +1,6 @@
 from user import User
 from board import Board
-from start import start_game 
+from start import GameModes, begin_game 
+from unittest.mock import patch
+from unittest import TestCase
```

Comment:
Lot of work to do here. Start simple. You've got these pieces now figure out how to put them together in a way that's easy to change.